tests/js/index.js

```diff
@@ -68,6 +68,7 @@ if (global.enableSyncTests) {
         TESTS.UserTests = require("./user-tests");
         TESTS.SessionTests = require("./session-tests");
         TESTS.UUIDSyncTests= node_require("./uuid-sync-tests");
+        TESTS.PartitionValueTests = node_require("./pv-tests");
```

Comment:
I would personally prefer a filename without an abbreviation, a newcomer to the project wouldn't be able to translate "pv" to "partition-value".

Reply:
agreed, changed.